### ISSUE TITLE
Confused about the compress rate of the history frame

### ISSUE THREAD

**Issue by DravenALG** [lllyasviel/FramePack#244]
From the codes in "demo_gradio.py" line 211:
`clean_latents_post, clean_latents_2x, clean_latents_4x = history_latents[:, :, :1 + 2 + 16, :, :].split([1, 2, 16], dim=2)`

and the codes in "diffusers_helper/models/hunyuan_video_packed.py" line 878-890:
```
if clean_latents_4x is not None and clean_latent_4x_indices is not None:
    clean_latents_4x = clean_latents_4x.to(hidden_states)
    clean_latents_4x = pad_for_3d_conv(clean_latents_4x, (4, 8, 8))
    clean_latents_4x = self.gradient_checkpointing_method(self.clean_x_embedder.proj_4x, clean_latents_4x)
    clean_latents_4x = clean_latents_4x.flatten(2).transpose(1, 2)

    clean_latent_4x_rope_freqs = self.rope(frame_indices=clean_latent_4x_indices, height=H, width=W, device=clean_latents_4x.device)
    clean_latent_4x_rope_freqs = pad_for_3d_conv(clean_latent_4x_rope_freqs, (4, 4, 4))
    clean_latent_4x_rope_freqs = center_down_sample_3d(clean_latent_4x_rope_freqs, (4, 4, 4))
    clean_latent_4x_rope_freqs = clean_latent_4x_rope_freqs.flatten(2).transpose(1, 2)

    hidden_states = torch.cat([clean_latents_4x, hidden_states], dim=1)
    rope_freqs = torch.cat([clean_latent_4x_rope_freqs, rope_freqs], dim=1)
```

It seems that the history frame from t-4 to t-20 using the same compress rate.

But in the paper, it proposes to compress the frame gradually by time. Why it is different in the code?

Am I understanding wrong or missing something? How to understand this operation?

**Comment by DravenALG** [comment]
I discuss this with my friend working in video generation. He also indicates that the method is implementing like this. 

But no matter how, it still gives many insights to this field.
